src/lib.rs

```diff
@@ -62,12 +62,18 @@
     anonymous_parameters,
     elided_lifetimes_in_paths
 )]
+#![allow(macro_expanded_macro_exports_accessed_by_absolute_paths)]
```

Comment:
This is necessary if we want to re-export the `html!` macro under the `macros` module or the `prelude` module. This will eventually not be allowed anymore so in the future, the only way to import the `html!` macro is with `use yew::html`. This is similar to how `serde` works, they require `use serde::{Deserialize, Serialize};`

cc @DenisKolodin

Reply:
If it works with stable compiler we can use it